src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,111 @@
+package duke;
+
+import duke.tasktypes.Deadlines;
+import duke.tasktypes.Events;
+import duke.tasktypes.Task;
+import duke.tasktypes.TaskList;
+import duke.tasktypes.ToDos;
+import java.io.BufferedWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Deals with loading tasks from the data file
+ * and saving tasks in the data file.
+ */
+public class Storage {
+
+    private String filePath;
+
+    public Storage(String filePath) {
```

Comment:
Should we add a description for this constructor (only asking because it has parameters)? I think its possible `filePath` could be explained in javadocs so that it can be used easily.